package.json

```diff
@@ -4,11 +4,9 @@
     "scripts": {
         "prepare": "lerna run prepare",
         "install-all": "npm install; lerna run prepare; lerna exec npm install",
-        "storybook:twig": "lerna exec --scope storybook npm run start:twig",
-        "storybook:react": "lerna exec --scope storybook npm run start:react",
-        "storybook:static": "npm run storybook:static:twig && npm run storybook:static:react",
-        "storybook:static:twig": "lerna exec --scope storybook npm run static:twig",
-        "storybook:static:react": "lerna exec --scope storybook npm run static:react",
+        "start": "lerna exec --scope storybook npm run start:react",
+        "storybook": "lerna exec --scope storybook npm run start:react",
+        "storybook static": "npm run storybook:static:react",
```

Comment:
I think this can just be `npm run storybook` or `npm start` since I think `start` and `storybook` are the same script